packages/web/src/common/store/pages/audio-transactions/sagas.ts

```diff
@@ -181,10 +177,7 @@ function* fetchTransactionsCount() {
         sdk.full.transactions,
         sdk.full.transactions.getAudioTransactionHistoryCount
       ],
-      {
-        encodedDataMessage: '', // TODO: remove, handled by sdk
-        encodedDataSignature: '' // TODO: remove, handled by sdk
-      }
+      {}
```

Comment:
oof this probably should be an optional param, but out of scope for this change probably.